central/sensor/service/pipeline/processlisteningonport/pipeline.go

```diff
@@ -56,12 +56,11 @@ func (s *pipelineImpl) Run(
 	if s.dataStore != nil && update != nil {
 		portProcesses := update.GetProcessesListeningOnPorts()
 
-		if portProcesses != nil {
```

Comment:
Running AddProcessListeningOnPort when portProcesses is nil is necessary so that old plops are retried even when there are no new plops added.